analytics-api/src/analytics_api/resources/survey_result.py

```diff
@@ -40,7 +40,30 @@ class SurveyResult(Resource):
     def get(engagement_id):
```

Comment:
auth optional or required for this one?

Reply:
ya..auth optional fills the jwt to our python context.. so we cant remove it..
and its an optional one since public can access the publsihed surveys without any token